### PR TITLE
fix: change wording for spec creation

### LINKED ISSUE #21908: Update 'create new empty spec' messaging.
## **Summary**
There’s an option for E2E and CT to ‘Create new empty spec’. But this doesn’t actually create an ‘empty’ specfile. It creates a file with some code for cy.visit or cy.mount. 

Screen Shot 2022-05-13 at 10 21 42 AM

Screen Shot 2022-05-13 at 10 14 26 AM

Screen Shot 2022-05-13 at 10 13 20 AM

## **Acceptance Criteria**
* We should update the wording of the ‘empty’ spec area. I’m not sure what it should say. 

┆Issue is synchronized with this [Jira Task](https://cypress-io.atlassian.net/browse/UNIFY-1759) by [Unito](https://www.unito.io)
┆Attachments: <a href="https://cypress-io.atlassian.net/rest/api/2/attachment/content/11978">Screen Shot 2022-05-13 at 10.13.20 AM.png</a> | <a href="https://cypress-io.atlassian.net/rest/api/2/attachment/content/11977">Screen Shot 2022-05-13 at 10.14.26 AM.png</a> | <a href="https://cypress-io.atlassian.net/rest/api/2/attachment/content/11976">Screen Shot 2022-05-13 at 10.21.42 AM.png</a>
┆author: Jennifer Shehane
┆epic: UI/UX Improvements
┆friendlyId: UNIFY-1759
┆priority: Medium
┆sprint: Backlog
┆taskType: Task

### PR DESCRIPTION
**Pull request by Manuel-Suarez-Abascal**
- Closes #21908

### User-facing changelog

Fix misleading wording when creating a new starter spec file in the UI.

### Additional details

N/A

### Steps to test

- Launch Cypress's test runner.
- Click on the button `New Spec` button.
- You should see the new wording in the cards to create a new starter spec file.

### PR Tasks

- [X] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

**Comment by lmiller1990** [comment]
Did you apply the requested changes from https://github.com/cypress-io/cypress/pull/24825 @Manuel-Suarez-Abascal?

Thanks, I triggered a CI run for you.

**Comment by Manuel-Suarez-Abascal** [comment]
> Did you apply the requested changes from #24825 @Manuel-Suarez-Abascal?
> 
> Thanks, I triggered a CI run for you.

👋🏻 
I have replied to Emily [here](https://github.com/cypress-io/cypress/pull/24825/files/cb11c3c3d520451f71dc5aadb59068dd41105f3a#r1056628681). Waiting for confirmation on her end.

**Comment by emilyrohrbough** [comment]
@Manuel-Suarez-Abascal Thanks for linking to your comment. I likely wouldn't have looked at the closed review. Glad you were able to get the CLA issue figured out!

File: packages/frontend-shared/src/locales/en-US.json [hunk 1]
```diff
@@ -111,8 +111,8 @@
         "chooseAComponentHeader": "Choose a component"
       },
       "importEmptySpec": {
-        "header": "Create a new spec",
-        "description": "We'll generate an empty spec file to start testing components.",
+        "header": "Create starter spec",
```
Comment (emilyrohrbough) [inline packages/frontend-shared/src/locales/en-US.json]: I think we should keep the header as `Create new spec`  and just update the description to use the updated term of starter spec. I don't have a super strong opinion on starter vs template -- I've just never heard the term starter before which is why I suggested template.

Comment (Manuel-Suarez-Abascal) [reply packages/frontend-shared/src/locales/en-US.json]: Sounds good, I'll make the changes suggested in the closed PR's comments & ping you guys when I'm done :)

**Comment by Manuel-Suarez-Abascal** [comment]
@lmiller1990 @emilyrohrbough I have addressed the PR comment. Let me know your thoughts or any other changes that I might have missed.

**Review by emilyrohrbough — APPROVED** [review]

**Comment by emilyrohrbough** [comment]
@Manuel-Suarez-Abascal looks like there a few failing tests with these changes.

**Review by lmiller1990 — APPROVED** [review]

**Comment by lmiller1990** [comment]
One of the component tests looks to be failing. Take a quick look @Manuel-Suarez-Abascal?

Thanks! Just do `cd packages/app && yarn cypress:run:ct --chrome` locally if you need to verify they all pass before pushing, they are pretty quick. Throw in `--headed` to watch them.

**Comment by Manuel-Suarez-Abascal** [comment]
> One of the component tests looks to be failing. Take a quick look @Manuel-Suarez-Abascal?
> 
> Thanks! Just do `cd packages/app && yarn cypress:run:ct --chrome` locally if you need to verify they all pass before pushing, they are pretty quick. Throw in `--headed` to watch them.

Thanks, @lmiller1990 I have to try to fix the error, but I'm not exactly sure why it's failing. I will need more time debugging it, but it doesn't find the `Create new spec` text on the screen which as you see in the screenshot below it's present. Thoughts 💭 ?

Screenshot 2022-12-29 at 8 05 08 AM

**Comment by Manuel-Suarez-Abascal** [comment]
@emilyrohrbough @lmiller1990 I have addressed the PR comments & fixed the failing spec files. I think it's ready for another review :)

Screenshot 2022-12-30 at 11 01 34 AM

**Pull request merged**